lightkurve/io/tests/test_pathos.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from astropy.io import fits
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from ... import search_lightcurve
+from ..pathos import read_pathos_lightcurve
+from .detect import detect_filetype
```

Comment:
Oops, you'll want to add a dot in front of detect to drop down a directory:
```suggestion
from ..detect import detect_filetype
```